### PR TITLE
add auto-dimming of displays/LEDs after 10pm (default to EST)

### PR DESCRIPTION
**Pull request by mastoca**
Not sure of interest on these small changes. This uses and ntp server to get the time and based on TZ_STRING (it is in posix format).

With the time set/known, it dims the LEDs & display at night to 5 (didn't notice any difference lower).

Thank you for the wonderful devices and project. I hope this helps you somehow.

EDIT: also modified the display's cosmetics a tad bit.

### CODE REVIEW

File: examples/ONE_V9/ONE_V9.ino [hunk 1]
```diff
@@ -205,10 +223,51 @@ void setup() {
...
+    previousbrightnessAdj += brightnessAdjInterval;
+    struct tm timeinfo;
+    if (!getLocalTime(&timeinfo)) {
+      Serial.println("Failed to obtain time");
+      return;
+    }
+    char timeHour[3];
+    strftime(timeHour, 3, "%H", &timeinfo);
+    String stringValue(timeHour);
+    int hour = stringValue.toInt();
```
Comment (Manezki) [inline examples/ONE_V9/ONE_V9.ino:262]: I believe this could be simplified to:
```suggestion
    int hour = timeinfo.tm_hour;
```

**Comment by Manezki** [comment]
Cheers man.

Your code saved me a couple of hours of tinkering.

I don't know about the PR practices of this repository, but it might be a good idea to remove the display cosmetics changes if you hope for the PR to be merged. I don't have privileges to merge your code, but I left a suggestion to simplify your code. I hope you don't mind.

Additionally, it appears you had tried to use `pixels.setBrightness()`. Did that not work for you, or did you observe some other drawbacks? Works fine in my case, so I wonder if you know something that I do not know.

**Comment by mastoca** [comment]
> Cheers man.
> 
> Your code saved me a couple of hours of tinkering.

Cheers!

> Additionally, it appears you had tried to use `pixels.setBrightness()`. Did that not work for you, or did you observe some other drawbacks? Works fine in my case, so I wonder if you know something that I do not know.

The issue I was referring to was the **oled** didn't get as dim as I wanted even with a value of `1`  but otherwise both the oled and led strip are dimmer and more subdued in the dark.

**Comment by pnt325** [comment]
Hi @mastoca 
Currently we don't have support dimming LED but you're able to turn it's off from cloud configure. You can get the newest firmware to get support at #80 

image

**Pull request closed without merging**